src/app/features/group/single-group/single-group.service.ts

```diff
@@ -0,0 +1,14 @@
+import { HttpClient } from '@angular/common/http';
+import { inject, Injectable } from '@angular/core';
+import { API_URL } from '@core/env.token';
+import { SingleGroup } from '../group.interface';
+
+@Injectable()
+export class SingleGroupService {
+  private http = inject(HttpClient);
+  private apiUrl = inject(API_URL);
+
+  getGroupInfo(groupId: string) {
+    return this.http.get<SingleGroup>(this.apiUrl + `/group/${groupId}`);
+  }
+}
```

Comment:
dlaczego nie tworzymy singletona skoro to serwis, który nie trzyma żadnego stanu?